src/xlora/xlora_insertion.py

```diff
@@ -104,7 +104,7 @@ def forward(self, x: Tensor, *args: Any, **kwargs: Any) -> Tensor:
                 scaling = self.target.scaling[active_adapter]
                 x = x.to(lora_A.weight.dtype)  # type: ignore
                 x_mod = self.apply_scalings_to_x(x, xlora_scalings, adapter_n)
-                result += lora_B(lora_A(dropout(x_mod))) * scaling
+                result += lora_B(lora_A(dropout(x_mod))) * scaling * self.config.global_scaling_weight
```

Comment:
All 3 are the same; we have 1 xLoraConfig for a xLoraModel, so the updating fn will update all.